devsetup/edpm/edpm-play.yaml

```diff
@@ -172,6 +172,9 @@ spec:
                 edpm_ovn_controller_image: quay.io/tripleomastercentos9/openstack-ovn-controller:current-tripleo
                 gather_facts: false
                 enable_debug: false
+                # edpm firewall, change the allowed CIDR if needed
+                edpm_sshd_configure_firewall: true
```

Comment:
Where are these variables implemented ? I don't see these in the files we have in this repo.

Reply:
They are needed by the nftables module that we want to import, see [this](https://github.com/openstack-k8s-operators/edpm-ansible/pull/29) PR.

Reply:
Sorry I totally missed the link.